addon-test-support/backstop.js

```diff
@@ -206,10 +206,28 @@ function validateName(name) {
   }
 }
 
+/**
+ * Reads environment.js config from the meta tag in test page, and returns the config object related to this addon.
+ */
+function getAddonCfgFromParentApp() {
+  const appEnvConfig = document.querySelector('meta[name*="config/environment"]').getAttribute('content');
+  return JSON.parse(decodeURIComponent(appEnvConfig))['ember-backstop'];
+}
+
 /**
  * I'm in your webapps -- checkin your screenz. -schooch
  */
 export default async function (assert, options) {
+  const { enableBackstop } = getAddonCfgFromParentApp();
```

Comment:
I would prefer to use `disableBackstop` here instead.  Then, we can test for `true` condition and existing implementations won't break.